projects/Douglas Mentoring/day1.py

```diff
@@ -0,0 +1,92 @@
+import getpass
+import netmiko
+# json was here bc as I'm using textfsm to pull data from cisco devices
+# it comes in json i needed to show the output before deciding how to tread the date
+import json
+import csv
+
+# Where i'm pulling the devices list / Soon I'll make this automated which will pull direct from NARF
+file = open(r'/home/segattod/segattod-workspace/src/Segattod-package/myproject/Light_level_checks/list.txt',
+            'r')
+file = file.read().splitlines()
+
+username = input('User Name:')
+password = getpass.getpass()
```

Comment:
As a general rule, these should be inside a method.  There isn't really any reason to have commands outside of a method.  Save the top up here for declaring constants, and everything else gets wrapped up tidy.